### ISSUE TITLE
ENH: Fontawesome loading improvements

### ISSUE THREAD

**Issue by Timo-Breumelhof** [DNNCommunity/Dnn.CommunityForums#410]
## Is your feature request related to a problem?
**Please describe.**
The Module loads the FontAwsome  icons CSS in the module's HTML this leads to a few problems:
- The module does not use the DNN CRM but a direct link inside the HTML , which work but is not valid HTML.
Also this means that it potentially conflict with FA loaded by the Theme (which is what I noticed for the new community Theme)

![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/8999382/ba22f0c1-625d-454a-bc02-867961626ba2)

- We load an old version of FA which also has more potential conflicting issues.
- There's no way to aet you don't want to load FA (when the Theme already loads it)

## Describe the solution you'd like
1. Use the DNN CRM to load FA (with the priority for ModuleCss = 10)
2. Use the latest version of FA 4 (5 or 6 are not really needed IMO)

## Describe alternatives you've considered
Offloading loading these to the Template / Theme?

## Additional context
- DNN: 09.11.00
- Module: 08.00.00
- 
https://github.com/DNNCommunity/Dnn.CommunityForums/blob/a810d9f65583ddd6438e7613d4aaac333beb9d10/Dnn.CommunityForums/Classic.ascx#L9

**Comment by Timo-Breumelhof** [comment]
I guess we could do the same for Bootstrap in case it's loaded by the Theme?

**Comment by johnhenley** [comment]
@Timo-Breumelhof if the theme is using dnncssinclude tag, and the module uses CRM, is the module setting really required? Won't CRM "figure it out" and only load one?

**Comment by Timo-Breumelhof** [comment]
Hmm, I guess so. I'll test it

**Comment by Timo-Breumelhof** [comment]
@johnhenley at least the order of the style sheet already fixed the icons "not loading" which was my most important point.

**Comment by Timo-Breumelhof** [comment]
@johnhenley , yes you are right. I think most Theme developers forget about this, but I now have only FA 5 and the v4 shim.

Theme:

```
        <!-- FontAwsome 5 + 4 shim-->
        <dnn:DnnCssInclude runat="server" FilePath="vendors/fontawesome-free-5.15.4-web/css/all.css" Name="fontawesome" Version="5.15.4" Priority="15" PathNameAlias="SkinPath" />
        <dnn:DnnCssInclude runat="server" FilePath="vendors/fontawesome-free-5.15.4-web/css/v4-shims.min.css" Name="fontawesome-shim4" Version="5.15.4" Priority="15" PathNameAlias="SkinPath" />
```

**Comment by Timo-Breumelhof** [comment]
https://github.com/DNNCommunity/Dnn.CommunityForums/wiki/Dependencies

**Comment by Timo-Breumelhof** [comment]
@johnhenley I removed the module settings requirement

**Comment by johnhenley** [comment]
[like]  John Henley reacted to your message:
________________________________

**Comment by johnhenley** [comment]
@Timo-Breumelhof sorry to keep asking questions but you changed the cdn to use cloudflare. While I'm sure that's likely to work for most sites, wouldn't it be better to use bootstrapcdn since that is the authoritative/"official" cdn referenced in the fontawesome documentation?

**Comment by Timo-Breumelhof** [comment]
> @Timo-Breumelhof sorry to keep asking questions but you changed the cdn to use cloudflare. While I'm sure that's likely to work for most sites, wouldn't it be better to use bootstrapcdn since that is the authoritative/"official" cdn referenced in the fontawesome documentation?

haha no problem. 
Sure I van change it to the BS option if you prefer.

**Comment by johnhenley** [comment]
> > @Timo-Breumelhof sorry to keep asking questions but you changed the cdn to use cloudflare. While I'm sure that's likely to work for most sites, wouldn't it be better to use bootstrapcdn since that is the authoritative/"official" cdn referenced in the fontawesome documentation?
> 
> 
> 
> haha no problem. 
> 
> Sure I van change it to the BS option if you prefer.

It's up to you guys. I just assumed cloudflare is a commercial entity and better to use an "open source"/ free cdn?

**Comment by Timo-Breumelhof** [comment]
Hmm, that last Commit does not seem to show up correctly in dev.. @johnhenley
